### PR TITLE
Twiml

### PR DESCRIPTION
**Pull request by jmatth**
Lots of refactoring (mostly just renaming stuff), and finished implementing the "Primary" TwiML verbs.

### CODE REVIEW

**Comment by natebrennand** [comment]
lgtm assuming all the magic XML reflection is working.
Want to try the examples again now that goconvey doesn't break?

File: twiml/noun_sip.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+type SipOpts struct {
+	Url      string `xml:"url,attr,omitempty"`
+	Method   string `xml:"method,attr,omitempty"`
+	Username string `xml:"username,attr,omitempty"`
+	Password string `xml:"password,attr,omitempty"`
+}
+
+func addSip(t twimlResponse, opts *SipOpts, sips []string) {
+	for _, uri := range sips {
+		t.appendContents(&sip{0, opts, uri})
```
Comment (natebrennand) [inline twiml/noun_sip.go]: missed one

**Comment by jmatth** [comment]
Oops, that just made me realize that the example snippet is out of date now. Also, goconvey does still crash when I have an example with expected output.

**Comment by jmatth** [comment]
Strike that, the example is up to date, I just had an old vim swap file or something that made it look out of date. Goconvey crashing is still an issue.

**Comment by natebrennand** [comment]
did you update goconvey? i.e. run `go get ..` again

**Pull request merged**